testharness/src/test/java/bio/terra/policy/service/policy/MutatorTest.java

```diff
@@ -58,5 +61,23 @@ void groupConstraintTest() throws Exception {
     Assertions.assertTrue(groupSet.contains(DDGROUP));
     Assertions.assertTrue(groupSet.contains(MCGROUP));
     Assertions.assertTrue(groupSet.contains(YUGROUP));
+
+    // remove dd from a group of 3
+    PolicyInput nodd = policyGroup.remove(mcyuddgroupPolicy, ddgroupPolicy);
+    groupSet = policyGroup.dataToSet(nodd.getAdditionalData().get(GROUP));
+    assertEquals(2, groupSet.size(), "Contains 2 groups");
```

Comment:
optional - These assertions can be simplified

```
assertThat(groupSet, containsInAnyOrder(MCGROUP, YUGROUP));
```

Reply:
Cool! Thanks!!